python/GafferUI/SpreadsheetUI.py

```diff
@@ -1588,15 +1617,28 @@ def __nodeEditorToolMenu( nodeEditor, node, menuDefinition ) :
 	assert( not ( selectorValue and selectorContextVariablePlug ) )
 
 	menuDefinition.append( "/SpreadsheetDivider", { "divider" : True } )
+
+	itemsActive = (
+		not nodeEditor.getReadOnly()
+		and not Gaffer.MetadataAlgo.readOnly( node )
+		and not Gaffer.MetadataAlgo.readOnly( activeRowNamesConnection )
+	)
+
 	menuDefinition.append(
 		"/Create Spreadsheet...",
 		{
 			"command" : functools.partial( __createSpreadsheetForNode, node, activeRowNamesConnection, selectorContextVariablePlug, selectorValue ),
-			"active" : (
-				not nodeEditor.getReadOnly()
-				and not Gaffer.MetadataAlgo.readOnly( node )
-				and not Gaffer.MetadataAlgo.readOnly( activeRowNamesConnection )
-			)
+			"active" : itemsActive
+		}
+	)
+
+	driveCommand = functools.partial( __drivePlugFromRowNames, activeRowNamesConnection, selectorContextVariablePlug, selectorValue )
+	menuDefinition.append(
+		"/Drive from Spreadsheet",
+		{
+			# "active" doesn't have any effect on subMenus
+			"subMenu" :  functools.partial( __spreadsheetSubMenu, activeRowNamesConnection, driveCommand ) if itemsActive else None,
+			"active" : itemsActive
```

Comment:
We need to disable this menu item when activeRowNamesConnection already has an input value.

Reply:
I had left that out as we weren't previously disabling `Create Spreadsheet...`. Thought that might have been intentional as a way to move the Node to a new spreadsheet?

Reply:
I think that was just a failure on my part. Wherever else we have menu items for making new connections, we disable them if there's an existing connection (expressions, randomisation, animation are all examples where we do this).

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3560/commits/53c8399b925d74e1a904654d45ee2cf559c13581